content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -99,6 +99,237 @@ Printer`. Follow the steps and select the *make* and *model* corresponding to th
+
+If the :abbr:`POS (Point of Sale)` system is using one of these printer models, a particular
+process must be undertaken in order to use another command; `ESC *`.
+
+Process to force ESC * command
+******************************
+
+The first step is to check whether the printer is incompatible with `GS v 0` command.
+
+.. seealso::
+   See this `documentation on Epison's site
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=94>`_ for compatible
+   printers.
+
+Should the printer not be compatible with the `GS v 0` command ensure that it is
+compatible with the `ESC *` command.
+
+.. seealso::
+   See the `Epson documentation
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=88>`_ for compatible
+   printers.
+
+If the printer is not compatible with the `ESC *` command then the following process is
+not possible. Should the printer be compatible to use the `ESC *` command to print, follow this
+process to configure the printer with the :abbr:`IoT (Internet of Things)` box.
+
+To manipulate the :abbr:`IoT (Internet of Things)` box to use the `ESC *` command to print, go to
+the :abbr:`IoT (Internet of Things)` box homepage. Access the :abbr:`IoT (Internet of Things)` box
+homepage by entering the :abbr:`IP (Internet Protocol)` address of the :abbr:`IoT (Internet of
+Things)` box into a browser window from a computer on the same network as the :abbr:`IoT (Internet
+of Things)` box (preferably by ethernet connection).
+
+.. note::
+   The :abbr:`IP (Internet Protocol)` address can be accessed by the router admin console that the
+   :abbr:`IoT (Internet of Things)` box is connected to, or by connecting a working receipt printer
+   to the :abbr:`IoT (Internet of Things)` box. A receipt will print out with the :abbr:`IoT
+   (Internet of Things)` box's :abbr:`IP (Internet Protocol)` address on it.
+
+Then click on the :guilabel:`Printers server` button. This will redirect the browser to the *CUPS
+page*. Next, go to :menuselection:`Administration --> Printers --> Add Printer`. Now choose the
+printer that should be modified. It could be listed as :guilabel:`Unknown`. Click
+:guilabel:`Continue`.
+
+.. tip::
+   If the name of the printer is still uncertain, take the following steps:
+
+   #. Take note of the listed printers on the *CUPS* page.
+   #. Turn the printer off and refresh the page.
+   #. Now compare the difference with the first list to see which printer disappeared.
+   #. Turn the printer back on and refresh the page again.
+   #. Double check the list again to see if the printer re-appears.
+   #. The printer that disappeared and reappears again on the listed printers in the name of the
+      printer in question.
+
+   .. note::
+      This can be :guilabel:`Unknown` under local printers.
+
+`CUPS` will prompt the admin for three pieces of information: the :guilabel:`Name`,
+:guilabel:`Description` and the :guilabel:`Location`. The last to pieces of information do not need
```

Comment:
"The last to pieces of information" redundant "to"

Reply:
or probably mean "two" ?

Reply:
Exactly, thank you! Made the change!